portal/raw-loader.ts

```diff
@@ -0,0 +1,4 @@
+declare module "!raw-loader*" {
+    const content: string;
+    export default content;
+}
```

Comment:
Using this feature requires `isolatedModules` to be turned off (which I did).

I'm having 2nd doubts if this is a good idea.

Reply:
It's prettier and easier to use. I like it. I think i like it more than the trade off... but maybe it should be documented somewhere

Reply:
We will be rewriting the portal pages soon™, so we can turn `isolatedModules` back on when we redo the example system anyway. Then again, we could also argue that we can just have the `@ts-ignore`s in there until then instead 🤷‍♂️

Reply:
But will the rewrite remove the need for raw import? I think we should keep showing the definition files in the New And Improved Portal, and yeah we can do that some other way, but for now that is by raw import. The rest should be rather easy to get rid of though.

Reply:
Yeah, you're right that it's not a given!